python/ray/train/callbacks/logging.py

```diff
@@ -236,8 +236,8 @@ def __init__(self,
         self.save_artifact = save_artifact
         self.mlflow_util = MLflowLoggerUtil()
 
-    def start_training(self, logdir: str, config: Dict, **info):
```

Comment:
Why are we removing `config` from here? It's needed for when we do `self.mlflow_util.log_params` down below

Reply:
Yeah sorry, that was a mistake, good catch!